internal/cmd/provider/create_version.go

```diff
@@ -0,0 +1,165 @@
+package provider
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/shurcooL/graphql"
+	"github.com/urfave/cli/v2"
+
+	"github.com/spacelift-io/spacectl/internal/cmd/authenticated"
+)
+
+func createVersion() cli.ActionFunc {
+	return func(cliCtx *cli.Context) error {
+		// Assuming that spacectl is ran from the root of the repository,
+		// containing the release artifacts in the "dist" directory.
+		dir := "dist"
```

Comment:
Should this maybe be a default value for the flag? This way, it would be automatically visible in usage print.

Reply:
Done